.editorconfig

```diff
@@ -0,0 +1,32 @@
+root = true
+
+[*]
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
```

Comment:
*All* files seems a bit aggressive, no? Not better to do this on a per-file-category basis?